v1.1 depends on moment

Can we please not depend on this horrible bloat?

https://github.com/you-dont-need/You-Dont-Need-Momentjs

I can't even upgrade to v1.1 because it breaks the build for some random reason. Plus the bundle gets bloated.

```
Module not found: Error: Can't resolve './locale' in '/var/www/html/apps/memories/node_modules/moment/min'
 @ ../node_modules/@nextcloud/moment/dist/index.mjs 1:0-50 2:0-65 6:0-8 9:58-72 9:73-81 15:0-17:2
 @ ../node_modules/@nextcloud/upload/dist/chunks/ConflictPicker-qrLWQ_8R.mjs 11:0-34 189:17-18
 @ ../node_modules/@nextcloud/upload/dist/chunks/index-dMIU3rSK.mjs 1668:20-59
 @ ../node_modules/@nextcloud/upload/dist/index.mjs 1:0-109 3:0-12:2 3:0-12:2 3:0-12:2 3:0-12:2 3:0-12:2 3:0-12:2 3:0-12:2 3:0-12:2
 @ ../node_modules/ts-loader/index.js??clonedRuleSet-4.use[0]!../node_modules/vue-loader/lib/index.js??vue-loader-options!./components/modal/UploadModal.vue?vue&type=script&lang=ts 19:0-48 176:29-40
 @ ./components/modal/UploadModal.vue?vue&type=script&lang=ts 1:0-193 1:209-212 1:214-404 1:214-404
 @ ./components/modal/UploadModal.vue 2:0-62 3:0-57 3:0-57 10:2-8
 @ ../node_modules/ts-loader/index.js??clonedRuleSet-4.use[0]!../node_modules/vue-loader/lib/index.js??vue-loader-options!./App.vue?vue&type=script&lang=ts 22:0-60 63:8-19
 @ ./App.vue?vue&type=script&lang=ts 1:0-173 1:189-192 1:194-364 1:194-364
 @ ./App.vue 2:0-54 3:0-49 3:0-49 10:2-8
 @ ./main.ts 4:0-28 40:21-24
```

--- a/lib/components/NodesPicker.vue
+++ b/lib/components/NodesPicker.vue
@@ -22,7 +22,14 @@
 				<!-- Description -->
 				<span class="node-picker__desc">
 					<span class="node-picker__name">{{ t('New version') }}</span>
-					<span class="node-picker__mtime">{{ lastModified(incoming) }}</span>
+					<NcDateTime v-if="incomingLastModified"
+						:timestamp="incomingLastModified"
+						:relative-time="false"
+						:format="{ timeStyle: 'short', dateStyle: 'medium' }"
+						class="node-picker__mtime" />
+					<span v-else class="node-picker__mtime">
+						{{ t('Last modified date unknown') }}
+					</span>
 					<span class="node-picker__size">{{ size(incoming) }}</span>
 				</span>
 			</span>
@@ -48,7 +55,14 @@
 				<!-- Description -->
 				<span class="node-picker__desc">
 					<span class="node-picker__name">{{ t('Existing version') }}</span>
-					<span class="node-picker__mtime">{{ lastModified(existing) }}</span>
+					<NcDateTime v-if="existingLastModified"
+						:timestamp="existingLastModified"
+						:relative-time="false"
+						:format="{ timeStyle: 'short', dateStyle: 'medium' }"
+						class="node-picker__mtime" />
+					<span v-else class="node-picker__mtime">
+						{{ t('Last modified date unknown') }}
+					</span>
 					<span class="node-picker__size">{{ size(existing) }}</span>
 				</span>
 			</span>
@@ -62,10 +76,10 @@ import type { PropType } from 'vue'
 import { defineComponent } from 'vue'
 import { formatFileSize, FileType, Node } from '@nextcloud/files'
 import { generateUrl } from '@nextcloud/router'
-import moment from '@nextcloud/moment'
 
 import FileSvg from 'vue-material-design-icons/File.vue'
 import FolderSvg from 'vue-material-design-icons/Folder.vue'
+import NcDateTime from '@nextcloud/vue/dist/Components/NcDateTime.js'
 import NcCheckboxRadioSwitch from '@nextcloud/vue/dist/Components/NcCheckboxRadioSwitch.js'
 
 import { t } from '../utils/l10n.ts'
@@ -79,6 +93,7 @@ export default defineComponent({
 		FileSvg,
 		FolderSvg,
 		NcCheckboxRadioSwitch,
+		NcDateTime,
 	},
 
 	props: {
@@ -102,7 +117,7 @@ export default defineComponent({
 
 	data() {
 		return {
-			asyncPreview: null,
+			asyncPreview: null as string | null,
 		}
 	},
 
@@ -127,17 +142,21 @@ export default defineComponent({
 		existingPreview() {
 			return this.previewUrl(this.existing)
 		},
+
+		incomingLastModified() {
+			return this.lastModified(this.incoming)
+		},
+		existingLastModified() {
+			return this.lastModified(this.existing)
+		},
 	},
 
 	methods: {
-		lastModified(node: File|Node): string {
+		lastModified(node: File|Node): Date | null {
 			const lastModified = node instanceof File
 				? new Date(node.lastModified)
 				: node.mtime
-			if (lastModified) {
-				return moment(lastModified).format('LLL')
-			}
-			return t('Last modified date unknown')
+			return lastModified ?? null
 		},
 		size(node: File|Node): string {
 			if (node.size) {
@@ -147,7 +166,7 @@ export default defineComponent({
 		},
 		previewUrl(node: File|Node) {
 			if (node instanceof File) {
-				this.previewImage(node).then((url: string) => {
+				this.previewImage(node).then((url: string | null) => {
 					this.asyncPreview = url
 				})
 				return

--- a/package.json
+++ b/package.json
@@ -76,7 +76,6 @@
     "@nextcloud/files": "^3.1.1",
     "@nextcloud/l10n": "^2.2.0",
     "@nextcloud/logger": "^2.7.0",
-    "@nextcloud/moment": "^1.3.1",
     "@nextcloud/paths": "^2.1.0",
     "@nextcloud/router": "^3.0.0",
     "axios": "^1.6.8",

